### PR TITLE
Adds ability to find a human-readable named string from a Pandora ID

### PR DESCRIPTION
**Pull request by jfy133**

### CODE REVIEW

File: R/dataprep_enforce_types.R [hunk 1]
```diff
@@ -10,51 +10,58 @@
...
+    suppress_na_introduced_warnings = suppress_na_introduced_warnings
+  )
   
   return(x)
 }
 
 #### helpers ####
 
+apply_var_types <- function(var_data, var_name, suppress_na_introduced_warnings) {
+  res <- var_data
```
Comment (jfy133) [inline R/dataprep_enforce_types.R:26]: Is this necessary? You only use it `var_trans_function()` in the if-else?

**Comment by jfy133** [comment]
Oh shit yeah. Forgot we decided that, sorry (running on 4h sleep)! The table I made is completed now, if you were waiting for that.

**Comment by nevrome** [comment]
Ok - I would like to do some pretty significant refactoring in the lookup and conversion functions. Are you OK with that?

**Comment by jfy133** [comment]
Go for it

**Comment by nevrome** [comment]
Thanks I'll try my best. We should have talked about a good structure more early.

**Pull request merged**